services/storage/object/controls.yaml

```
@@ -84,13 +84,17 @@ controls:
     test_requirements:
       - id: CCC.ObjStor.C03.TR01
         text: |
-          Object storage buckets cannot be deleted after creation.
+          When an object storage bucket is created then it must be protected from all deletion attempts, including both hard delete and soft delete, to ensure irrevocable data integrity and prevent any loss.
+        tlp_levels:
+          - tlp_red
+      - id: CCC.ObjStor.C03.TR02
+        text: |
+          When an object storage bucket is created, then soft delete may be allowed, but it must enforce a retention period that allows for recovery of deleted buckets within the specified timeline to prevent unintended permanent data loss.
```

Brainstorming... feel free to iterate on this:

 ```suggestion
          When an object storage bucket deletion is attempted, the bucket MUST be fully recoverable for a set timeframe after deletion is requested.
```